docs/modules/getting_started/partials/con_differences-from-production-openshift-install.adoc

```diff
@@ -8,6 +8,7 @@ Upgrading the {ocp} version might cause issues that are difficult to reproduce.
 * It uses a single node, which behaves as both a control plane and worker node.
 * It disables the Cluster Monitoring Operator by default.
 This disabled Operator causes the corresponding part of the web console to be non-functional.
+* It does not include the Cloud Credential Operator.
 * The {ocp} cluster runs in a virtual machine known as an __instance__.
```

Comment:
there are more cluster operators that are currently disabled maybe that should also be listed, from: https://github.com/crc-org/snc/blob/master/cvo-overrides.yaml

- cluster-autoscaler
- cloud-controller-manager

Reply:
@anjannath Yup, done.

Reply:
A lot of repetition in tone/structure.

Perhaps best to use a list here:

* It disables the following operators:
  - Cloud Credential Operator
  - Cluster Autoscaler Operator
  - Cluster Cloud Controller Manager Operator
  - Cluster Monitoring Operator

@themr0c WDYT?

Reply:
@gbraad I find it somewhat challenging to include the Cluster Monitoring Operator in the same list as the other three. Unlike the others, it is only disabled 'by default' (can be enabled day 1 if needed) and its disablement results in the corresponding section of the web console becoming non-functional. 

How about this:
```adoc
* The Cluster Monitoring Operator is disabled by default. This results in the corresponding part of the web console to be non-functional.
* The following operators are always disabled:
** Cloud Credential Operator.
** Cluster Autoscaler Operator.
** Cluster Cloud Controller Manager Operator.
```